nixos/modules/services/web-servers/caddy/default.nix

```diff
@@ -41,6 +41,12 @@ let
     in
       "${if pkgs.stdenv.buildPlatform == pkgs.stdenv.hostPlatform then Caddyfile-formatted else Caddyfile}/Caddyfile";
 
+  etcConfigFile = "caddy/caddy_config";
+
+  configPath = if cfg.enableReload
```

Comment:
For simplicity we should just always move the config file to /etc

Reply:
I agree with this.